docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md

```diff
@@ -71,10 +71,20 @@ With this attestation service design, we can avoid adding malicious applications
 
 #### Attestation Service for Cloud Service Provider (CSP)
 
```

Comment:
Add a sentence:
Azure provides an Attestation Service for applications running in TEE VM or containers provided by Azure.